templates/scenarios/js/non-sso-tab-default-bot/.vscode/tasks.json

```diff
@@ -0,0 +1,126 @@
+// This file is automatically generated by Teams Toolkit.
+// The teamsfx tasks defined in this file require Teams Toolkit version >= 4.1.0.
+// See https://aka.ms/teamsfx-debug-tasks for details on how to customize each task and how to integrate with existing Teams Toolkit projects.
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Start Teams App Locally",
+            "dependsOn": [
+                "Validate prerequisites",
+                "Start local tunnel",
+                "Create resources",
+                "Set up local projects",
+                "Start services"
+            ],
+            "dependsOrder": "sequence"
+        },
+        {
+            // Check if all required prerequisites are installed.
+            // See https://aka.ms/teamsfx-check-prerequisites-task to know the details and how to customize the args.
+            "label": "Validate prerequisites",
+            "type": "teamsfx",
+            "command": "debug-check-prerequisites",
+            "args": {
+                "prerequisites": [
+                    "nodejs", // Validate if Node.js is installed.
+                    "m365Account", // Sign-in prompt for Microsoft 365 account, then validate if the account enables the sideloading permission.
+                    "portOccupancy" // Validate available ports to ensure those debug ones are not occupied.
+                ],
+                "portOccupancy": [
+                    53000 ,// tab service port
+                    3978, // bot service port
+                    9239 // bot inspector port for Node.js debugger
+                ]
+            }
+        },
+        {
+            // Start the local tunnel service to forward public ngrok URL to local port and inspect traffic.
+            // See https://aka.ms/teamsfx-local-tunnel-task to know the details and how to customize the args.
+            "label": "Start local tunnel",
+            "type": "teamsfx",
+            "command": "debug-start-local-tunnel",
+            "args": {
+                "ngrokArgs": "http 3978 --log=stdout --log-format=logfmt",
+                "env": "local"
```

Comment:
Please apply the latest change in https://github.com/OfficeDev/TeamsFx/pull/6889.